### DISCUSSION TITLE
How to get the drive ID?

### DISCUSSION THREAD

**Discussion by elhennig** [abraunegg/onedrive#2780] [General]
Hi, I try to get the drive ID from a newly created sharepoint library, but the output only show the names of the available libraries. How do I run the command correctly?

**Comment by abraunegg** [comment]
@elhennig 
The driveID is in the output ........ as per the documentation: 

```
Configuration file successfully loaded
Configuring Global Azure AD Endpoints
Initializing the Synchronization Engine ...
Office 365 Library Name Query: <your site name to search>
-----------------------------------------------
Site Name:    <your site name>
Library Name: <your library name>
drive_id:     b!6H_y8B...xU5
Library URL:  <your library URL>
-----------------------------------------------
```

So .. if your are not getting the above output .. what application version are you using?

**Reply by elhennig** [reply]
OK, I used the url to the site instead of the name. Probably that was the mistake I did.
